TbsCore/Helpers/MapHelper.cs

```diff
@@ -133,46 +132,46 @@ public static List<MapTile> GetMapTiles(Account acc, Coordinates coords)
                 //    return mapPositionData.tiles.Select(x => x.GetMapTile()).ToList();
                 //}
                 default:
-                {
-                    var bearerToken = DriverHelper.GetBearerToken(acc);
-
-                    var reqMapInfo = new RestSharp.RestRequest
                     {
-                        Resource = "/api/v1/map/info",

```

Comment:
why was this removed?

Reply:
huh why on earth this is removed ?

Reply:
ah ic
It is because my VS extension "beautify" the code and make git confused, 
if u watch In normal view u can see it just like the previous